pydantic_ai_slim/pydantic_ai/models/__init__.py

```diff
@@ -382,6 +390,14 @@ async def request(
         """Make a request to the model."""
         raise NotImplementedError()
 
+    @abstractmethod
```

Comment:
Since this method is not required to be implemented, we can do the same thing we do in `request_stream`, meaning not mark it as `@abstractmethod` (so we can drop all the empty implementations from the model classes) and put the `Token counting is not supported by <X>` error message here.